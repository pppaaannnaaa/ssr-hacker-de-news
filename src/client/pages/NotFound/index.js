import React from 'react';
// import PropTypes from 'prop-types';

const NotFound = () => {
  return (
    <div className="ui container" role="banner" aria-label="Error Page">
      <h1>Page Not Found!!!</h1>
      <p>Please try again!</p>
    </div>
  );
};

NotFound.propTypes = {};

NotFound.defaultProps = {};

export default {
  component: NotFound,
};
