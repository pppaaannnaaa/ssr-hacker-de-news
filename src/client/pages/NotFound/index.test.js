import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import NotFound from './index';

describe('<NotFound />', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  afterEach(cleanup);

  test('Renders <NotFound /> component correctly', async () => {
    // eslint-disable-next-line react/jsx-pascal-case
    const { container } = render(<NotFound.component />);
    expect(container).toMatchSnapshot();
  });
});
