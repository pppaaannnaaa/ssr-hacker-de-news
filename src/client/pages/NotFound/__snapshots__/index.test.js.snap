// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Renders <NotFound /> component correctly 1`] = `
<div>
  <div
    aria-label="Error Page"
    class="ui container"
    role="banner"
  >
    <h1>
      Page Not Found!!!
    </h1>
    <p>
      Please try again!
    </p>
  </div>
</div>
`;
